src/Sylius/Bundle/PayumBundle/Extension/UpdatePaymentStateExtension.php

```diff
@@ -26,7 +23,6 @@ class UpdatePaymentStateExtension implements ExtensionInterface
      * @var FactoryInterface
      */
     protected $factory;
-
```

Comment:
Sadly, these all removed blank lines should be reverted, due to coding standards principles.